backend/src/controllers/middlewaresControllers/createAuthMiddleware/authUser.js

```diff
@@ -2,7 +2,7 @@ const bcrypt = require('bcryptjs');
 const jwt = require('jsonwebtoken');
 
 const authUser = async (req, res, { user, databasePassword, password, UserPasswordModel }) => {
-  const isMatch = await bcrypt.compare(databasePassword.salt + password, databasePassword.password);
+  const isMatch = await bcrypt.compare(password, databasePassword.password);
```

Comment:
I believe we should stick to the existing hashing logic [https://github.com/idurar/idurar-erp-crm/blob/master/backend/src/controllers/middlewaresControllers/createAuthMiddleware/authUser.js](here) otherwise admin login will fail

Reply:
Okay I will adjust this. Thanks for bringing this into view.